rules/prevent-abbreviations.js

```diff
@@ -178,6 +180,16 @@ const defaultWhitelist = {
 	stdDev: true
 };
 
+const getCase = string => {
+	for (const fn of [camelCase, kebabCase, snakeCase, upperfirst]) {
```

Comment:
Shouldn't this use `pascalCase` instead of `upperfirst`?

Reply:
will send another PR to fix it